NEWS.rst

```diff
@@ -10,6 +10,25 @@ Updates
 
 - max line length from 80 to 120
 - Rewrite context.read method using COPY TO (#570)
+- Add new visualization API (#662)
+  - Add Source object (with param detection)
```

Comment:
`class` instead of `object` (I'm sorry to be so annoying)

Reply:
:+1: Your are right